tests/CMakeLists.txt

```diff
@@ -1,14 +1,3 @@
+#add_subdirectory(yarpWholeBodyModelTest)
```

Comment:
I removed it because I need to update the CMakeLists.txt for the
wholeBodyModelTest.

---
Naveen Kuppuswamy, PhD
Post-doctoral Fellow,
Cognitive Humanoids Lab,
Department of Robotics, Brain and Cognitive Sciences (RBCS),
Istituto Italiano di Tecnologia, Genova, Italy

On Fri, Feb 20, 2015 at 11:59 AM, Silvio Traversaro <
notifications@github.com> wrote:

> In tests/CMakeLists.txt
> <https://github.com/robotology-playground/yarp-wholebodyinterface/pull/23#discussion_r25062807>
> :
>
> > @@ -1,14 +1,3 @@
> > +#add_subdirectory(yarpWholeBodyModelTest)
>
> I guess we should leave yarpWholeBodyModel test enabled.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robotology-playground/yarp-wholebodyinterface/pull/23/files#r25062807>
> .
>